spec/features/collection_spec.rb

```diff
@@ -240,7 +240,7 @@
     visit collection_path(@collection.owner, @collection)
     expect(page).to have_content("About")
     expect(page).to have_content("Works")
-    page.click_link("Pages That Need Transcription")
+    page.click_link("Pages That Need Correction or Transcription")
```

Comment:
Can we externalize these messages?  We'll need a way to mark them in the dictionary file as "needing translation" -- maybe with a comment that the translator can remove once it's translated?